tests/garage/sampler/test_utils.py

```diff
@@ -23,11 +23,6 @@ def test_max_path_length(self):
         assert path['agent_infos']['dummy'].shape[0] == 3
         assert path['env_infos']['dummy'].shape[0] == 3
 
-    def test_does_flatten(self):
```

Comment:
Wait, shouldn't this test still pass? Or maybe it's named wrong.

Reply:
no this test checks to see that the sampler is flattening observations. After a discussion we had, the conclusion we came to was that the sampler shouldn't be doing any flattening or unflattening.

Reply:
perhaps you should just modify it to `test_does_not_flatten` instead